.gitlab/build_quartz.yml

```diff
@@ -57,9 +57,9 @@ clang_10_0_0 (PR build_on_quartz):
     HOST_CONFIG: "quartz-toss_3_x86_64_ib-${COMPILER}.cmake"
   extends: .pr_build_on_quartz
 
-gcc_8_3_1 (PR build_on_quartz):
+gcc_8_1_0 (PR build_on_quartz):
```

Comment:
@hennis1 why are we going back to 8.1.0? At some point in the near future, gcc 8.3.1 will be the default on LC systems.

Reply:
Because we have never had 8.3.1.  This came from Serac's file.  We can update this after we work out the kinks in the process.

Reply:
> Because we have never had 8.3.1. This came from Serac's file. We can update this after we work out the kinks in the process.

OK. Thanks